### DISCUSSION TITLE
feyncalc for python

### DISCUSSION THREAD

**Discussion by HosseinF90** [FeynCalc/feyncalc#197] [Q&A]
Hello.
How can i use feyncalc in python codes?
is there any chance for accessing this package through python?

**Comment by vsht** [comment]
Hi,

I don't really have any experience with that, but if you find a way to access Mathematica
from Python, you could also load FeynCalc and use it this way.

Perhaps this can be useful:

https://reference.wolfram.com/language/WolframClientForPython/
https://mathematica.stackexchange.com/questions/218729/how-to-run-a-mathematica-package-m-from-python-using-the-wolfram-client-libr
